src/server/identity/cmds/cmds.go

```diff
@@ -6,6 +6,11 @@ import (
 	"io/ioutil"
 	"os"
 
+	"github.com/gogo/protobuf/types"
+
+	"github.com/ghodss/yaml"
+	"github.com/pachyderm/pachyderm/v2/src/server/identity/server"
```

Comment:
I think it would be better here to have a shared identityutil package that has the PickConfig function, rather than pull the server into the client